clients/drcachesim/tracer/raw2trace.h

```diff
@@ -80,10 +81,11 @@
 #define WINDOW_SUBDIR_FIRST "window.0000"
 #define TRACE_SUBDIR "trace"
 #ifdef HAS_ZLIB
-#    define TRACE_SUFFIX "trace.gz"
```

Comment:
I was going to say that is not supported -- but I ended up later making it not a fatal error to not initialize the submodule, so this can happen.  It is not ideal and I hesitated there: we won't have good tests of all this different configs and maybe it's better to not claim we support this.  OTOH I remember issues with having missing submodules causing fatal errors.

Reply:
Added .trace.gz back for !HAS_ZIP